politeiawww/codetracker/github/database/database.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package database
+
+import (
+	"errors"
+)
+
+var (
+	// ErrNoVersionRecord is emitted when no version record exists.
+	ErrNoVersionRecord = errors.New("no version record")
+
+	// ErrNoPullRequestFound is emitted when no pull request matches.
+	ErrNoPullRequestFound = errors.New("no pull request found")
+
+	// ErrWrongVersion is emitted when the version record does not
+	// match the implementation version.
+	ErrWrongVersion = errors.New("wrong version")
+
+	// ErrShutdown is emitted when the cache is shutting down.
+	ErrShutdown = errors.New("cache is shutting down")
+
+	// ErrUserNotFound indicates that a user name was not found in the
+	// database.
+	ErrUserNotFound = errors.New("user not found")
+)
+
+// Database interface that is required by the web server.
+type Database interface {
```

Comment:
Clean this interface up.  Don't use inline comments for method declaration comments.  It makes it hard to read.